Dockerfile

```diff
@@ -1,9 +1,6 @@
-# Building on Ubuntu Bionic (18.04) and supporting glibc 2.27. This allows
-# the following distros (and newer versions) to run the resulting binaries:
-# * Ubuntu Bionic (2.27)
-# * Debian Buster (2.28)
-# * CentOS 8 (2.28)
-FROM ubuntu:bionic as builder
+FROM ubuntu:20.04 as builder
```

Comment:
Any reason to change this?

Reply:
There will be errors like when use lower LLVM version:
```
bindsnoop.bpf.c:76:20: error: use of unknown builtin '__builtin_preserve_access_index' [-Wimplicit-function-declaration]
.output/bpf/bpf_core_read.h:404:2: note: expanded from macro 'BPF_CORE_READ'
        BPF_CORE_READ_INTO(&__r, (src), a, ##__VA_ARGS__);                  \
```

Reply:
Let's use https://apt.llvm.org/bionic/ if Ubuntu repos don't have what we need.